examples/official-storybook/stories/parameters.examples.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+// We would need to add this in config.js idomatically however that would make this file a bit confusing
+import { addParameters } from '@storybook/react';
+
+const globalParameter = 'globalParameter';
+const chapterParameter = 'chapterParameter';
+const storyParameter = 'storyParameter';
+
+addParameters({ globalParameter });
+
+export default {
+  title: 'Core|Parameters',
+  parameters: {
+    chapterParameter,
+  },
+};
+
+// I'm not sure what we should recommend regarding propTypes? are they a good idea for examples?
+// Given we sort of control the props, should we export a prop type?
+// eslint-disable-next-line react/prop-types
+export const passed = ({ parameters: { fileName, ...parameters } }) => (
```

Comment:
need to think now what to do with the `addon-storysource` 🤔

Reply:
indeed